theme/common.less

```diff
@@ -1713,6 +1713,11 @@ div.simframe.ui.embed {
     div.description {
         line-height: 1.1em;
         margin-bottom: 20px;
+
+        .warning {
```

Comment:
don't we have css for standard warnings? Is this special somehow?

Reply:
Yes, probably, but this warning message is temporary and will be removed once the feature is out of experiment. If it were going to be long-lived, I would for sure want to use a standard css class.